app/controllers/application_controller.rb

```diff
@@ -222,8 +222,7 @@ def set_default_url_for_mailer
     end
   end
 
-  def person_belongs_to_current_community
-    @person = Person.find(params[:person_id] || params[:id])
+  def ensure_person_belongs_to_current_community!(person)
```

Comment:
Since this is no longer a filter I don't see much reason to keep it in ApplicationController. Why not separate view util for person-related controller utility functions similarly as we have e.g. PaypalHelper?